src/main/java/fi/oph/akt/repository/AuthorisationTermRepository.java

```diff
@@ -14,4 +15,23 @@
 			+ " FROM AuthorisationTerm at")
 	List<AuthorisationTermProjection> listAuthorisationTermProjections();
 
+	// @formatter:off
+	@Query("SELECT at"
+			+ " FROM AuthorisationTerm at"
+			+ " LEFT JOIN AuthorisationTermReminder atr ON atr.authorisationTerm.id = at.id"
+			+ " WHERE at.endDate >= ?1 AND at.endDate <= ?2"
```

Comment:
Onko `WHERE at.endDate BETWEEN ?1 AND ?2` sama asia? https://openjpa.apache.org/builds/1.2.3/apache-openjpa/docs/jpa_langref.html#jpa_langref_between